app/services/omniauth_bimedoc_service.rb

```diff
@@ -0,0 +1,22 @@
+class OmniauthBimedocService < BaseOmniauthService
+  def initialize(omniauth_info)
+    Rails.logger.debug(omniauth_info.to_json) # TODO REMOVE THIS DEBUG
+    @omniauth_info = omniauth_info
+    @sub = omniauth_info.sub
+    @provider_id = "bimedoc"
+    @service_name = "Bimedoc"
+  end
+
+  def info
+    {
+      provider_id: provider_id,
+      service_name: service_name,
+      sub: sub,
+      # TODO remove omniauth_info DEBUG
+      DEBUG_RAW_DATA: omniauth_info
```

Comment:
:question: Does that means this should be removed now or later?

Reply:
Bimedoc ne fonctionne pas actuellement, il manque des trucs a synchro entre eux et nous, d'où le debug.